app/build.gradle

```diff
@@ -212,7 +214,8 @@ dependencies {
     implementation "com.yarolegovich:discrete-scrollview:1.4.9"
     implementation "com.appyvet:materialrangebar:1.4.4"
     implementation "com.github.antonygolovin:fluentsnackbar:1.0.0"
-    implementation "com.github.adrielcafe:PageIndicatorView:1.0.6" // original library doesn't support ViewPager2
+    implementation "com.github.adrielcafe:PageIndicatorView:1.0.6"
+    // original library doesn't support ViewPager2
```

Comment:
Why is this comment affected?

Reply:
This changes appear automatically when i commit the file since my reformat code option is enabled